build failure for v1.0.4

http://mobile.jenkins.couchbase.com/job/couchbase-lite-java-build/50/console

```
/home/couchbase/jenkins/workspace/couchbase-lite-java-build/couchbase-lite-java/libraries/couchbase-lite-java-core/src/main/java/com/couchbase/lite/support/Version.java:44: error: unreachable statement
        return -1;
```

--- a/src/main/java/com/couchbase/lite/support/Version.java
+++ b/src/main/java/com/couchbase/lite/support/Version.java
@@ -11,7 +11,7 @@ public class Version {
     static {
         int versCode = getVersionCode();
         if (versCode == -1) {
-            VERSION = String.format("%s-%s", getVersionName(), getVersionCode());
+            VERSION = String.format("%s-%s", getVersionName(), getVersion());
         } else{
             VERSION = String.format("%s", getVersionName());
         }

